win32ss/gdi/ntgdi/freetype.c

```diff
@@ -903,6 +903,7 @@ IntGdiLoadFontsFromMemory(PGDI_LOAD_FONT pLoadFont,
     /* set face */
     FontGDI->SharedFace = SharedFace;
     FontGDI->charset = ANSI_CHARSET;
+    FontGDI->codepage = 1252;
```

Comment:
Add a comment about what CP this number corresponds to? (windows-1252 | ANSI Latin 1; Western European (Windows).)

Reply:
OK.